### PR TITLE
PRESIDECMS-1769 impersonate indication on admin toolbar

### PR DESCRIPTION
**Pull request by mookeshNanda**
**Hi @DominicWatson ,**

Have added indication on admin toolbar when impersonating. 
Followed @john5onCheng instruction.
Homepage

Please do accept this.
https://projects.pixl8.london/browse/BDA-405

**Thanks & Regards**
Mookesh Nanda

### CODE REVIEW

File: system/coldboxModifications/RequestContextDecorator.cfc [hunk 1]
```diff
@@ -402,6 +402,10 @@ component extends="coldbox.system.web.context.RequestContextDecorator" {
 		return _getSticker().includeUrl( argumentCollection = arguments );
 	}
 
+	public boolean function isImpersonated() {
```
Comment (DominicWatson) [inline system/coldboxModifications/RequestContextDecorator.cfc]: Can this be named `isWebUserImpersonated()` please? (we will at some point add admin user impersonation)

File: system/views/general/_adminToolbar.cfm [hunk 2]
```diff
@@ -94,6 +94,9 @@
 					</cfif>
 					<div class="navbar-header pull-right">
 						<ul class="nav ace-nav">
+							<cfif #event.isImpersonated()#>
```
Comment (DominicWatson) [inline system/views/general/_adminToolbar.cfm]: The `#`'s around this are redundant:

`<cfif event.isWebUserImpersonated()>`

File: system/views/general/_adminToolbar.cfm [hunk 3]
```diff
@@ -94,6 +94,9 @@
...
+								<li>&nbsp; <label class="text-success">Impersonating</label> : #getLoggedInUserDetails().email_address#&nbsp;</li>
```
Comment (DominicWatson) [inline system/views/general/_adminToolbar.cfm]: This should be `i18n`:

```cfm
<li>&nbsp; <i class="fa fa-fw fa-user-md green"></i> #translateResource( uri="cms:admin.bar.impersonating.web.user", data=[ getLoggedInUserDetails().display_name ]  )#</li>
```

**Review by DominicWatson — CHANGES_REQUESTED** [review]
Hi @mookeshNanda thanks for this. I've left some review feedback. Could you take a look and implement?

File: system/views/general/_adminToolbar.cfm (hunk 3 above)
Comment (mookeshNanda) [reply system/views/general/_adminToolbar.cfm]: Hi @DominicWatson ,
Could we use email address in order to maintain the uniqueness.

**Comment by DominicWatson** [comment]
Brill, thanks for the changes @mookeshNanda :). Will be released in 10.11.25

**Pull request merged**